doc/ref_model/chapters/appendix-a.md

```diff
@@ -26,3 +27,15 @@ The goal is not to be prescriptive on how to re-architect existing or architect
 
 <a name="A.4"></a>
 ## A.4 Links
+
+
+<a name="A.5"></a>
+## A.5 Miscellaneous
+### A.5.1 VNF Monitoring Capabilities - UseCase.
```

Comment:
I agree that current description is more towards VNF network troubleshooting area. Updated the title accordingly.
@petorre : Updated description to specifically mention that it is NFVI platform is responsible for exposing it.

Reply:
Ashok, This is good. It would be great if you could put some more narrative around using the feature, the scenario, etc. 

To that point, you mentioned leveraging a vTAP, but you also mentioned the NFVI would have to expose the capability. I can see the vTAP approach if the monitoring is intra-VNF. However, if the monitoring is being performed inside the monitored VNF itself (intra-VNF), an Infra-based monitoring capability is not required and the solution should be implemented at the discretion of the VNF designer, not CNTT. 

If the monitoring is inter-VNF, which I suspect is more likely, I would expect the mirrored traffic to be exposed to the monitoring VNF on a typical virtual interface, bubbling up from the Infra and ingressing the relevant VNF, the same as standard pt-to-pt traffic. What network topology/scenario were you envisioning with w/ the vTAP implementation? Thanks, -Mark

Reply:
@markshostak : Thanks. I have explained now how NFVI must provide capability to expose mirrored traffic on new virtual interface so that monitoring VNF can use it.
I currently do not see any other monitoring topology other then pt-to-pt as of now.

Reply:
Thank you. I'll take the liberty and Resolve this thread for Kelvin. @kedmison if you disagree, ping me and let's discuss. Thanks, -Mark